spec/features/reservations_spec.rb

```diff
@@ -552,7 +552,6 @@
       select('10', from: 'items_per_page') # show more than 1 item (default)
```

Comment:
Ah yeah I didn't think about it from an efficiency perspective--will implement this now